content/operate/rc/compatibility.md

```diff
@@ -1,17 +1,17 @@
 ---
-Title: Redis Cloud compatibility with source available Redis
+Title: Redis Cloud compatibility with Redis Community Edition
 alwaysopen: false
 categories:
 - docs
 - operate
 - rc
-description: Redis Cloud compatibility with source available Redis.
+description: Redis Cloud compatibility with Redis Community Edition.
 linkTitle: Source available compatibility
```

Comment:
```suggestion
linkTitle: Community Edition compatibility
```

Reply:
Thank you, @cmilesb ! Can you modify your suggestion to include Redis?

linkTitle: Redis Community Edition compatibility

Reply:
@dwdougherty Done!